lib/child_process.js

```diff
@@ -94,17 +87,17 @@ function normalizeExecArgs(command /*, options, callback*/) {
 }
 
 
-exports.exec = function(command /*, options, callback*/) {
-  var opts = normalizeExecArgs.apply(null, arguments);
+exports.exec = function(command, options, callback) {
+  const opts = normalizeExecArgs(command, options, callback);
   return exports.execFile(opts.file,
                           opts.options,
                           opts.callback);
 };
 
 
-exports.execFile = function(file /*, args, options, callback*/) {
-  var args = [], callback;
-  var options = {
+exports.execFile = function(file, args, options, callback) {
+
+  const defaultOptions = {
```

Comment:
Would there be any benefit to pulling this out of the function and into the top level module scope?